diesel/src/types/mod.rs

```diff
@@ -396,6 +438,7 @@ pub trait FromSqlRow<A, DB: Backend + HasSqlType<A>>: Sized {
     /// the number of times you would call `row.take()` in `build_from_row`
```

Comment:
I think transform one of the places `FromSql` into a link would improve the trait documentation. (Because if someone implements this he will go forth and back between the `FromSql` and `FromSqlRow` documentation)